test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```
@@ -27,6 +27,11 @@ set(TEST_SCRIPT_FILES
     cpp/testFlowViewportAPIFilterPrims.py
 )
 
+#Add this test only if the MAYAUSDAPI_LIBRARY has been found during compile time.
+if(MAYAUSDAPI_LIBRARY)
```

The reason is because `MAYAUSDAPI_LIBRARY` is a required variable so it must be set when `MayaUsd_FOUND` is true.
```suggestion
if(MayaUsd_FOUND)
```